docs/streams/index.html

```diff
@@ -21,7 +21,7 @@ <h1>Kafka Streams API</h1>
           <div class="sticky-top">
              <div style="height:35px">
                 <a  class="active-menu-item" href="/{{version}}/documentation/streams/">Introduction</a>
-                <a href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
+                <a href="/{{version}}/documentation/streams/developer-guide/">Developer Guide</a>
```

Comment:
Does this auto-redirect to `../developer-guide/index.html`?

Reply:
@derrickdoo ^^

i have so far been unable to get this working on my local build, so I do not know the answer.

Reply:
@guozhangwang yes, it does redirect